src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -24,27 +32,37 @@ specific language governing permissions and limitations under the License.
 }
 
 .tab.selected {
-  background: var(--theme-tab-active-background);
+  background-position: var(--theme-tab-active-background-position);
+  box-shadow: var(--theme-focus-outline-style);
   color: var(--theme-tab-active-color);
 }
 
 .tab.deselected {
-  background: var(--theme-tab-inactive-background);
-  box-shadow: 1px 0 0 0 var(--theme-tab-separator-color);
-  color: var(--theme-tab-inactive-color);
+  /* react-css-modules generateAppendClassName requires that this exist */
 }
 
 .tab.initial {
   width: var(--theme-tab-initial-width);
+  height: var(--theme-tab-initial-height);
+  background-position: var(--theme-tab-inactive-background-position);
   opacity: var(--theme-tab-initial-opacity);
 }
 
 .tab.open {
-  width: var(--theme-tab-open-width);
+  height: var(--theme-tab-open-height);
   opacity: var(--theme-tab-open-opacity);
 }
 
 .tab.closed {
   width: var(--theme-tab-closed-width);
   opacity: var(--theme-tab-closed-opacity);
+  transition: width 0.5s var(--animation-curve);
+}
+
+.tab.selected-closed {
+  width: var(--theme-tab-closed-width);
+  height: var(--theme-tab-closed-height);
+  background-position: var(--theme-tab-active-background-position);
```

Comment:
As for what `background-position` is doing there, if it is removed we get two animations going. when closing the selected tab, the first is the tab height shrinking which is desired. the second, since `.selected` has been removed, the background will shift to show the default colour. You can see this in the animation if you slow it down,  eventually the background will go lower than the text on the tab.